src/config/user.cr

```diff
@@ -48,7 +55,11 @@ struct DppmRestApi::Config::User
   end
 
   def self.deserialize(groups : String)
-    groups.split(',').map { |id| id.to_i base: 16 }
+    Set(Int32).new.tap do |set|
```

Comment:
I don't see the need to make a tap block here, just do `set = Set(Int32).new` and an indent is saved